extension/intents/scroll/scrollHelper.js

```diff
@@ -2,8 +2,13 @@
 
 const scrollAmount = 0.9;
 
+const url = location.href;
+const pdf = url.slice(-4) === ".pdf";
+const activeEl = document.activeElement;
```

Comment:
Note that `.activeElement` can change while a page is loaded (it follows focus), so this should be calculated each time.